### PR TITLE
GEODE-6588: Cleanup static analyzer warnings.

### PR DESCRIPTION
**Pull request by pivotal-jbarrett**
### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### CODE REVIEW

**Review by onichols-pivotal — COMMENTED** [review]
_comment deleted_

**Review by onichols-pivotal — COMMENTED** [review]
Thanks for improving your commit message!  Don't forget, first commit additional detail paragraph(s) should describe the problem and how this change solves it.

File: geode-wan/src/main/java/org/apache/geode/cache/wan/internal/client/locator/LocatorHelper.java [hunk 1]
```diff
@@ -12,11 +12,12 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
+
```
Comment (boglesby) [inline geode-wan/src/main/java/org/apache/geode/cache/wan/internal/client/locator/LocatorHelper.java:15]: Why did you add the space here (and in LocatorMembershipListenerImpl below)? I don't see other files with a space between the banner and package.

Comment (pivotal-jbarrett) [reply geode-wan/src/main/java/org/apache/geode/cache/wan/internal/client/locator/LocatorHelper.java:15]: I do it sometimes... habit... Like the separation of the unrelated comment block from the symbol. 🤷

**Review by boglesby — APPROVED** [review]
